server/src/main/java/org/opensearch/cluster/metadata/View.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.metadata;
+
+import org.opensearch.cluster.AbstractDiffable;
+import org.opensearch.cluster.Diff;
+import org.opensearch.common.annotation.ExperimentalApi;
+import org.opensearch.core.ParseField;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ConstructingObjectParser;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+/** View of data in OpenSearch indices */
+@ExperimentalApi
+public class View extends AbstractDiffable<View> implements ToXContentObject {
+
+    private final String name;
+    private final String description;
+    private final long createdAt;
+    private final long modifiedAt;
```

Comment:
Do we need these? 

I did a search for `createdAt` in OpenSearch and it doesn't seem to be a recurring pattern. Everything else that we create just "is".

Reply:
Hmm, its pretty inconsistent.

- RepositoriesStatsArchive.ArchiveEntry has createdAtMillis
- TranslogTransferMetadata has createdAt
- BulkByScrollTask.Status has created

Personally I would prefer to keep `createdAt` which implies its a time value, opposed to `created` which could imply a time value or the entity that created the object.

Did my search by looking for created[a-zA-Z]*; in the codebase, maybe I missed some clearer examples?

Reply:
Oh! hah I orginally read your feedback as 'created' vs 'createdAt'.  I like to include a created/modified time to preserve a basic trail of "did someone change it and it broke?"  How strongly do you feel about this?